applications/fishing-map/src/features/timebar/Timebar.tsx

```diff
@@ -83,9 +96,7 @@ const TimebarWrapper = () => {
     (clientX: number, scale: (arg: number) => Date) => {
       if (hoverInEvent.current === false) {
         if (clientX === null || clientX === undefined || isNaN(clientX)) {
-          if (highlightedTime !== undefined) {
-            dispatch(disableHighlightedTime())
-          }
+          dispatch(disableHighlightedTime())
```

Comment:
I think this was necessary to avoid spamming the store, did you check it?

Reply:
It will set an already `undefined` value to `undefined`, so it shouldn't trigger any change on the store, correct?
How can we be 100% sure of this. This was needed to avoid having TimebarWrapper having a dependency on a 'high-frequency' prop.